test/simple/StatsPrinter.h

```diff
@@ -24,5 +24,8 @@ class StatsPrinter : public Stats {
   void frameWritten(const std::string& frameType) override;
   void frameRead(const std::string& frameType) override;
   void resumeBufferChanged(int framesCountDelta, int dataSizeDelta) override;
+
+  void connectionClosedInServerConnectionAcceptor(
```

Comment:
nit: this seems awfully specific.  Stats is really only for reporting on errors, could we use a more generic name.  badIncomingConnection?